app/views/jobs/show.html.erb

```diff
@@ -45,12 +43,15 @@
 
       <!-- When deleted you are sent back to the job index page -->
       <div class="edits text-center">
+        <%= link_to "Edit Rating", edit_job_path(@job) %>
```

Comment:
this is a link to edit job, is it the same for the rating because it's on the show page?